schemas/verifiablePresentation.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import verifiableCredential from './verifiableCredential.js';
+import {extend as _extend} from '../lib/helpers.js';
+import proof from './proof.js';
+import identifier from './identifier.js';
+import {klona} from 'klona';
+
+const schema = {
+  title: 'Verifiable Presentation',
+  type: 'object',
+  additionalProperties: true,
+  properties: {
+    '@context': {
+      type: 'array',
+      minItems: 1,
+      items: [
+        {type: 'string', const: 'https://www.w3.org/2018/credentials/v1'},
+        {anyOf: [{type: 'string'}, {type: 'object'}]}]
+    },
+    id: identifier(),
+    type: {
+      type: 'array',
+      minItems: 1,
+      items: {
```

Comment:
The first item has to be `VerifiablePresentation`.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-validation/pull/44/commits/f6ee4b99d158a51d4af4f9e80985df9dff0014fb